okhttp/src/main/java/com/squareup/okhttp/internal/http/HttpMethod.java

```diff
@@ -41,6 +41,11 @@ public static boolean permitsRequestBody(String method) {
         || method.equals("LOCK");     // (WebDAV) body: create lock, without body: refresh lock
   }
 
+  public static boolean redirectsToGET(String method) {
+    // all requests but propfind should redirect to a GET request
```

Comment:
will fix this but the lines above in the same file did not really adhere to that, therefore, I didn't know that this was in the code style.